pytket/docs/build-docs.sh

```diff
@@ -3,6 +3,8 @@ cp -R pytket-docs-theming/quantinuum-sphinx .
 cp pytket-docs-theming/conf.py .
 
 sphinx-build -b html . build -D html_title="pytket"
+find build/ -type f -name "*.html" | xargs sed -e 's/pytket._tket/pytket/g' -i ""
```

Comment:
slightly worried that these sed comands are not architecture independent. They work fine on my mac.

The website uses linux runners in C.I. so will make sure to use the linux syntax there.

Reply:
Could also check the platform from within the shell script and perform a switch on that.

Reply:
Can confirm that this does not work on Linux.